src/AbstractSQLRules2SQL.ts

```diff
@@ -452,7 +467,7 @@ export const checkMinArgs = (matchName: string, args: any[], num: number) => {
 const AddDateNumber: MatchFn = (args, indent) => {
 	checkArgs('AddDateNumber', args, 2);
 	const a = DateValue(getAbstractSqlQuery(args, 0), indent);
-	const b = NumericValue(getAbstractSqlQuery(args, 1), indent);
+	const b = mathOpValue(args, 1, indent);
```

Comment:
Only checking the right site operand is not sufficient. As postgres date functions explicit define the different outcome from operations bound to the left and right operant of date arithmetics:
https://www.postgresql.org/docs/current/functions-datetime.html